compiler/rustc_mir/src/borrow_check/region_infer/mod.rs

```diff
@@ -1241,7 +1241,7 @@ impl<'tcx> RegionInferenceContext<'tcx> {
     /// it. However, it works pretty well in practice. In particular,
     /// this is needed to deal with projection outlives bounds like
     ///
-    /// ```ignore (internal compiler representation so lifetime syntax is invalid)
+    /// ```text (internal compiler representation so lifetime syntax is invalid)
```

Comment:
There is no need for the explanation insde the paren, it's only required with `ignore`.